patches/chrome-installer-mini_installer-mini_installer.cc.patch

```diff
@@ -103,6 +103,24 @@ index 1f39abbeb2579098a54fa82612199f0307ea03b4..80095fc3ab546770adc098a36bef9e79
 +  if (*scan == L'.')
 +    filename.truncate_at(scan - filename.get());
 +
++  // Strip any de-duplicating suffix from filename, e.g. "(1)".
++  scan = filename.get() + filename.length() - 1;
++  if (*scan == L')') {
++    --scan;
++    while (scan != filename.get() && *scan >= '0' && *scan <= '9')
++      --scan;
++    if (*scan == L'(')
++      filename.truncate_at(scan - filename.get());
++  }
++
++  // Strip trailing spaces from filename.
++  scan = filename.get() + filename.length() - 1;
++  while (scan != filename.get() && *scan == L' ')
++    --scan;
++
++  if (scan != filename.get() && (scan != filename.get() + filename.length()))
++    filename.truncate_at(scan - filename.get() + 1);
++
```

Comment:
Just noting that the mini installer forbids any use of the C standard library, hence all of the pointer work here.